components/x-teaser/src/MetaLink.jsx

```diff
@@ -23,7 +23,8 @@ export default ({ metaPrefixText, metaLink, metaAltLink, metaSuffixText, context
 				<a
 					className="o-teaser__tag"
 					data-trackable="teaser-tag"
-					href={displayLink.relativeUrl || displayLink.url}>
+					href={displayLink.relativeUrl || displayLink.url}
+					aria-label="Category: {displayLink.prefLabel}">
```

Comment:
Because this is JSX we'll need the string interpolation to be slightly different here:
aria-label={`Category: ${displayLink.prefLabel}`}

The outside brackets tell the file to interpret this snippet as JavaScript which lets us use a template literal inside.